### PR TITLE
{vis}[intel/2021b] XCrySDen v1.6.2

### PR DESCRIPTION
**Pull request by lcniel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lcniel** [comment]
Test report by @lcniel
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in total)
vera-icelake-build - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Silver 4316 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/lcniel/f95cf38e039a97435d749ef21e316e8b for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/lcniel/0bcfe58c5756b7fe5b5924b75704835d for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
node041.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/e27a0e5566c84ad4f69a29a360e64913 for a full test report.

File: easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+homepage = "http://www.xcrysden.org/"
+docurls = "http://www.xcrysden.org/Documentation.html"
+description = """
+XCrySDen is a crystalline and molecular structure visualisation program aiming at display of isosurfaces
+and contours, which can be superimposed on crystalline structures and interactively rotated and manipulated.
+It also possesses some tools for analysis of properties in reciprocal space such as interactive selection
+of k-paths in the Brillouin zone for the band-structure plots, and visualisation of Fermi surfaces.
+"""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb:16]: I don't see this software using MPI or MKL, so could be dropped to `intel-compilers`

Comment (lcniel) [reply easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb]: OK, done, no strong opinions on this.

**Comment by lcniel** [comment]
Test report by @lcniel
**FAILED**
Build succeeded for 0 out of 1 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/lcniel/8784c33b62063c050943d3448d5c69e2 for a full test report.

File: easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb (hunk 1 above)
Comment (lcniel) [reply easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb]: @verdurin @jfgrimm Seems like intel-compilers fail due to a dependency on `fftw3.h`, if I am not mistaken that is an MKL dependency, so I don't think intel-compilers will work.

**Comment by lcniel** [comment]
Test report by @lcniel
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/lcniel/ae954b9d2dc2d8e5408001eb0ce333a5 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/lcniel/0a964195b398b3cae06f74d32942d3bc for a full test report.

File: easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb]: Yes it does dep on fftw3.h so must be at intel level, you need to rename the easyconfig to match the toolchain.

Comment (lcniel) [reply easybuild/easyconfigs/x/XCrySDen/XCrySDen-1.6.2-intel-2021b.eb:16]: That should take care of it.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/akesandgren/916e2aecbd54a275b6ac4bf56cb5ad6e for a full test report.

**Comment by akesandgren** [comment]
On second thought, it doesn't have to be any higher than at the iimkl level. unfortunately we currently don't have iimkl/2021b
So we'll move it down to that for any newer versions, i.e. no need to change this now.

**Comment by akesandgren** [comment]
Going in, thanks @lcniel!

**Pull request merged**